client/src/components/pages/LoginPage/style.css

```diff
@@ -0,0 +1,7 @@
+.login-page--main-section {
+    min-height: calc(100vh - calc(129px + 70px));
+    width: 80%;
```

Comment:
```suggestion
    width: 90%;
```

Reply:
@aminalakhsham  again
your mission to solve the issue of heigh